extensions/arc/src/actions.tsx

```diff
@@ -313,3 +313,14 @@ export function EditTabActionSection(props: { tab: Tab; mutate: MutatePromise<Ta
     </ActionPanel.Section>
   );
 }
+
+export function CreateQuickLinkActionSection(props: { url: string; title?: string }) {
+  return (
+    <ActionPanel.Section>
+      <Action.CreateQuicklink
+        quicklink={{ link: props.url, name: props.title, application: "Arc" }}
+        shortcut={{ modifiers: ["cmd"], key: "d" }}
```

Comment:
```suggestion
        shortcut={{ modifiers: ["cmd"], key: "s" }}
```

Reply:
I was inspired by the pin/bookmarks tab action of most browser: 
<img width="752" alt="image" src="https://user-images.githubusercontent.com/17577332/218783047-4f93da54-fa5d-4dca-80cd-4354d324e579.png">

Reply:
I committed the suggestion since it's not really related to pinning tabs (and for making some similarities to how other extensions are doing it)

Reply:
sounds good, let's merge this !